layouts/blog/single.html

```diff
@@ -1,37 +1,70 @@
-          {{ $page_parts := split .Page.Params.target_release "." }}
-          {{ $page_version := int (index $page_parts 0) }}
-          {{ $page_revision := int (index $page_parts 1) }}
-          {{ $page_normalized_version := add (add (mul $page_version 1000000) (mul $page_revision 1000)) 0 }}
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+  <div class="header-content">
+    <h1>{{ .Params.title }}</h1>
+    <p>{{ .Params.description }}</p>
+  </div>
+  <p class="post-author">
+    {{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{
+    replace (replace .Params.attribution "(" "- ") ")" "" }}
+  </p>
+   {{ if .Page.Params.target_release }}
+      {{ $page_parts := split .Page.Params.target_release "." }}
+      {{ $page_version := int (index $page_parts 0) }}
+      {{ $page_revision := int (index $page_parts 1) }}
+      {{ $page_normalized_version := add (add (mul $page_version 1000000) (mul $page_revision 1000)) 0 }}
 
-          {{ $site_parts := split .Site.Data.args.version "." }}
-          {{ $site_version := int (index $site_parts 0) }}
-          {{ $site_revision := int (index $site_parts 1) }}
-          {{ $site_normalized_version := add (add (mul $site_version 1000000) (mul $site_revision 1000)) 0 }}
+      {{ $site_parts := split .Site.Data.args.version "." }}
+      {{ $site_version := int (index $site_parts 0) }}
+      {{ $site_revision := int (index $site_parts 1) }}
+      {{ $site_normalized_version := add (add (mul $site_version 1000000) (mul $site_revision 1000)) 0 }}
 
-          {{ if gt $site_normalized_version $page_normalized_version }}
-              <div>
-                  <aside class="callout warning">
-                      <div class="type">
-                          {{- partial "large_icon.html" "callout-warning" -}}
-                      </div>
-                      <div class="content">
-                          {{ printf (i18n "target_release") .Page.Params.target_release }}
-                      </div>
-                  </aside>
-              </div>
-          {{ end }}
-      {{ end }}
-      <div>
-        {{ .Content }}
+      {{ if gt $site_normalized_version $page_normalized_version }}
+  <div>
+    <aside class="callout warning">
+      <div class="type">
+        {{- partial "large_icon.html" "callout-warning" -}}
+      </div>
+      <div class="content">
+        {{ printf (i18n "target_release") .Page.Params.target_release }}
       </div>
-      {{ partial "posts_navigation.html" . }}
-    </article>
-{{ end }}
\ No newline at end of file
+    </aside>
+  </div>
+  {{ end }} {{ end }}
+  <div>{{ .Content }}</div>
+    <p class="social-share__title"><h4>SHARE</h4></p>
+    <div class="social-share__wrapper">
+      <a
+        href="https://www.linkedin.com/shareArticle?mini=true&url={{ .Permalink }}"
+        target="_blank"
+      >
+        <img
+          src="/img/social/linkedin.svg"
+          style="width: 50px; height: 50px"
```

Comment:
Almost feel like this should be a class added to the blog SASS file instead of repeated here.